__tests__/app.test.js

```diff
@@ -190,4 +190,54 @@ describe('Testing app', () => {
           .expect(204);
         });
     });
+    describe('POST', () => {

```

Comment:
good tests!

consider

`POST /api/articles/9999/comments`
and
`POST /api/articles/not-an-id/comments`

Reply:
Added